src/platforms/react-native/upload-debug.mdx

```diff
@@ -0,0 +1,17 @@
+---
+title: Debug Symbols
+sidebar_order: 96
+description: "Learn about using ProGuard to upload the ProGuard/R8 mappings, source bundle, and native symbols Sentry."
+---
+
+Debug symbols provide information that Sentry displays on the **Issue Details** page to help you triage an issue. We offer a range of methods to provide Sentry with debug symbols.
+
+## When to Upload
+
+With default settings, complete stack traces are available in your Java/Kotlin error, out of the box, unless you use [enableProguardInReleaseBuilds](https://reactnative.dev/docs/signed-apk-android#enabling-proguard-to-reduce-the-size-of-the-apk-optional). In this case, you need to upload the ProGuard/R8 mapping file generated by the [Android Gradle Plugin](https://developer.android.com/studio/build/shrink-code#shrink-code), so Sentry can show you proper stack traces.
+
+Errors raised from the native layer in Android apps require certain debug information files to be uploaded. For example, an Android app can use `proguard` for minification and obfuscation. Also, when using [NDK](https://developer.android.com/ndk/), dwarf debug files need to be uploaded.
+
+## Uploading With ProGuard and NDK
+
+To use ProGuard or NDK with Sentry, upload the [Android Proguard/R8 mapping files and native symbols](/platforms/android/proguard/) by either the recommended method of using our Gradle integration or manually by using `sentry-cli` [upload-proguard](/product/cli/dif/#proguard-mapping-upload) or [upload-dif](/product/cli/dif/#uploading-files).
```

Comment:
@imatwawana maybe I should split this up into 2 sections, one of them is the Sentry Android Gradle plugin (/platforms/android/proguard/) and the other section with sentry-cli, (/product/cli/dif/) links, so that fits the "range of methods", what do you think?

Reply:
Depends on the answer to my other question about the main aim of this page.